autoware_launch/launch/logging_simulator.launch.xml

```diff
@@ -20,114 +21,45 @@
+      <!-- Common -->
       <arg name="vehicle_model" value="$(var vehicle_model)"/>
       <arg name="sensor_model" value="$(var sensor_model)"/>
-      <arg name="vehicle_id" value="$(var vehicle_id)"/>
-      <arg name="launch_vehicle_interface" value="$(var launch_vehicle_interface)"/>
-    </include>
-  </group>
-
-  <!-- System -->
-  <group>
-    <include file="$(find-pkg-share tier4_system_launch)/launch/system.launch.xml" if="$(var system)">
-      <arg name="run_mode" value="online"/>
-      <arg name="sensor_model" value="$(var sensor_model)"/>
-    </include>
-  </group>
-
-  <!-- Map -->
-  <group>
-    <include file="$(find-pkg-share tier4_map_launch)/launch/map.launch.py" if="$(var map)">
-      <arg name="lanelet2_map_path" value="$(var map_path)/$(var lanelet2_map_file)"/>
-      <arg name="pointcloud_map_path" value="$(var map_path)/$(var pointcloud_map_file)"/>
-    </include>
-  </group>
-
-  <!-- Sensing -->
-  <group>
-    <include file="$(find-pkg-share tier4_sensing_launch)/launch/sensing.launch.xml" if="$(var sensing)">
-      <arg name="launch_driver" value="false"/>
-      <arg name="sensor_model" value="$(var sensor_model)"/>
-      <arg name="vehicle_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.param.yaml"/>
-      <arg name="vehicle_mirror_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/mirror.param.yaml"/>
+      <!-- Global parameters -->
+      <arg name="use_sim_time" value="true"/>
+      <!-- Pointcloud container -->
       <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
       <arg name="pointcloud_container_name" value="$(var pointcloud_container_name)"/>
+      <!-- Vehicle -->
+      <arg name="launch_vehicle" value="$(var vehicle)"/>
+      <arg name="vehicle_id" value="$(var vehicle_id)"/>
+      <arg name="launch_vehicle_interface" value="$(var launch_vehicle_interface)"/>
+      <!-- System -->
+      <arg name="launch_system" value="$(var system)"/>
+      <arg name="system_run_mode" value="online"/>
+      <!-- Map -->
+      <arg name="launch_map" value="$(var map)"/>
+      <arg name="map_path" value="$(var map_path)"/>
+      <arg name="lanelet2_map_file" value="$(var lanelet2_map_file)"/>
+      <arg name="pointcloud_map_file" value="$(var pointcloud_map_file)"/>
+      <!-- Sensing -->
+      <arg name="launch_sensing" value="$(var sensing)"/>
+      <arg name="launch_sensing_driver" value="false"/>
+      <!-- Localization -->
+      <arg name="launch_localization" value="$(var localization)"/>
+      <!-- Perception -->
+      <arg name="launch_perception" value="$(var perception)"/>
+      <!-- Planning -->
+      <arg name="launch_planning" value="$(var planning)"/>
+      <!-- Control -->
+      <arg name="launch_control" value="$(var control)"/>
+      <!-- API -->
+      <arg name="init_simulator_pose" value="false"/>
+      <arg name="init_localization_pose" value="true"/>
```

Comment:
[IMO] Reordering the options would make it easy to understand. Like this? :thinking: 

```xml
      <!-- Common -->
      <arg name="map_path" value="$(var map_path)"/>
      <arg name="lanelet2_map_file" value="$(var lanelet2_map_file)"/>
      <arg name="pointcloud_map_file" value="$(var pointcloud_map_file)"/>

      <!-- Modules to be launched -->
      <arg name="launch_map" value="$(var map)"/>
```

Reply:
https://github.com/autowarefoundation/autoware_launch/pull/59/commits/e022dbd42db234b8d8a61ff2e8ddee94b54baf82
I don't understand 100%, but is this what you wanted to do?

Reply:
I meant the order of args wasn't organized well. But it seems now it's improved!
![image](https://user-images.githubusercontent.com/31987104/178663643-22fe166e-5b49-4a20-ada6-58d77f635364.png)